source/manual/covid-19-services.html.md

```diff
@@ -63,13 +64,14 @@ Both applications have a similar architecture of:
 - **CI:** GitHub Actions (branches and PRs)
 - **CD:** Concourse (master builds and production deploys)
 - **Logging:** Sentry and Logit
+- **Email**: GOV.UK Notify
 
```

Comment:
Do you think it would be worth adding Sidekiq to this list? We're running a mailers queue for the confirmation emails.

Reply:
I've added it here anyway.  Useful to have a summary of the full tech stack.